packages/matchbox/src/components/WindowEvent/tests/WindowEvent.test.js

```diff
@@ -9,35 +9,31 @@ describe('WindowEvent', () => {
   let wrapper;
   const handler = jest.fn();
 
-  beforeEach(() => {
-    wrapper = shallow(<WindowEvent event='scroll' handler={handler}/>);
-  });
+  const subject = () => mount(<WindowEvent event="scroll" handler={handler} />);
 
   afterEach(() => {
     jest.resetAllMocks();
   });
 
   it('renders nothing', () => {
+    wrapper = subject();
     expect(wrapper.html()).toEqual(null);
   });
 
   it('adds event listener on mount', () => {
-    expect(global.addEventListener).toHaveBeenCalledTimes(1);
```

Comment:
So I tried logging: `console.log(global.addEventListener.mock.calls);`

and got this:
```js
[ [ 'error', [Function: handleWindowError] ],
      [ 'error', [Function: handleWindowError] ],
      [ 'error', [Function: handleWindowError] ],
      [ 'scroll',
        { [Function: mockConstructor]
          _isMockFunction: true,
          getMockImplementation: [Function],
          mock: [Getter/Setter],
          mockClear: [Function],
          mockReset: [Function],
          mockReturnValueOnce: [Function],
          mockResolvedValueOnce: [Function],
          mockRejectedValueOnce: [Function],
          mockReturnValue: [Function],
          mockResolvedValue: [Function],
          mockRejectedValue: [Function],
          mockImplementationOnce: [Function],
          mockImplementation: [Function],
          mockReturnThis: [Function],
          mockName: [Function],
          getMockName: [Function],
          mockRestore: [Function] } ],
      [ 'error', [Function: handleWindowError] ],
      [ 'error', [Function: handleWindowError] ],
      [ 'error', [Function: handleWindowError] ] ]
```

Looks like the events are being called once correctly, but it looks like the testing setup is throwing global window errors from somewhere.. no idea